bioptim/limits/path_conditions.py

```diff
@@ -687,6 +700,18 @@ def concatenate(self, other: "InitialGuess"):
             interpolation=self.init.type,
         )
 
+    def scale(self, scaling: float) -> "InitialGuess":
+        """
+        Scaling an InitialGuess
+
+        Parameters
+        ----------
+        scaling: float
+            The scaling factor
+        """
+        self.init = self.init / scaling
+        return self
```

Comment:
This should not return anything

Reply:
I need it for a more straightforward way of writing :
https://github.com/fbailly/bioptim/blob/9d75595998618004794e2f1b83cd0315fdaca2a2/bioptim/optimization/parameters.py#L78